src/main.rs

```diff
@@ -11,9 +11,110 @@ use std::marker::PhantomData;
+    const seven: i64 = 7;
+    const eight: i64 = 8;
+    const nine: i64 = 9;
+    const ten: i64 = 10;
+    const eleven: i64 = 11;
+    const twelve: i64 = 12;
+    const fourteen: i64 = 14;
+    const fifteen: i64 = 15;
+    const sixteen: i64 = 16;
+    const seventeen: i64 = 17;
+    const eighteen: i64 = 18;
+    const nineteen: i64 = 19;
+    const twenty: i64 = 20;
+    const twentyone: i64 = 21;
+    const twentytwo: i64 = 22;
+    const twentythree: i64 = 23;
+    const twentyfour: i64 = 24;
+    const twentyfive: i64 = 25;
+    const twentysix: i64 = 26;
+    const twentyseven: i64 = 27;
+    const twentyeight: i64 = 28;
+    const twentynine: i64 = 29;
+    const thirty: i64 = 30;
+    const thirtyone: i64 = 31;
+    const thirtytwo: i64 = 32;
+    const thirtythree: i64 = 33;
+    const thirtyfour: i64 = 34;
+    const thirtyfive: i64 = 35;
+    const thirtysix: i64 = 36;
+    const thirtyseven: i64 = 37;
+    const thirtyeight: i64 = 38;
+    const thirtynine: i64 = 39;
+    const fourty: i64 = 40;
+    const fourtyone: i64 = 41;
+    const fourtytwo: i64 = 42;
+    const fourtythree: i64 = 43;
+    const fourtyfour: i64 = 44;
+    const fourtyfive: i64 = 45;
+    const fourtysix: i64 = 46;
+    const fourtyseven: i64 = 47;
+    const fourtyeight: i64 = 48;
+    const fourtynine: i64 = 49;
+    const fifty: i64 = 50;
+    const fiftyone: i64 = 51;
+    const fiftytwo: i64 = 52;
+    const fiftythree: i64 = 53;
+    const fiftyfour: i64 = 54;
+    const fiftyfive: i64 = 55;
+    const fiftysix: i64 = 56;
+    const fiftyseven: i64 = 57;
+    const fiftyeight: i64 = 58;
+    const fiftynine: i64 = 59;
+    const sixty: i64 = 60;
+    const sixtyone: i64 = 61;
+    const sixtytwo: i64 = 62;
+    const sixtythree: i64 = 63;
+    const sixtyfour: i64 = 64;
+    const sixtyfive: i64 = 65;
+    const sixtysix: i64 = 66;
+    const sixtyseven: i64 = 67;
+    const sixtyeight: i64 = 68;
+    const sixtynine: i64 = 69;
+    const seventy: i64 = 70;
+    const seventyone: i64 = 71;
+    const seventytwo: i64 = 72;
+    const seventythree: i64 = 73;
+    const seventyfour: i64 = 74;
+    const seventyfive: i64 = 75;
+    const seventysix: i64 = 76;
+    const seventyseven: i64 = 77;
+    const seventyeight: i64 = 78;
+    const seventynine: i64 = 79;
+    const eighty: i64 = 80;
+    const eightyone: i64 = 81;
+    const eightytwo: i64 = 82;
+    const eightythree: i64 = 83;
+    const eightyfour: i64 = 84;
+    const eightyfive: i64 = 85;
+    const eightysix: i64 = 86;
+    const eightyseven: i64 = 87;
+    const eightyeight: i64 = 88;
+    const eightynine: i64 = 89;
+    const ninety: i64 = 90;
+    const ninetyone: i64 = 91;
+    const ninetytwo: i64 = 92;
+    const ninetythree: i64 = 93;
+    const ninetyfour: i64 = 94;
+    const ninetyfive: i64 = 95;
+    const ninetysix: i64 = 96;
+    const ninetyseven: i64 = 97;
+    const ninetyeight: i64 = 98;
+    const ninetynine: i64 = 99;
+    const onehundred: i64 = 100;
```

Comment:
```suggestion
    const ninetyten: i64 = 100;
```

To keep consistency with numbers from 0 to 10.